rotkehlchen/tests/utils/globaldb.py

```diff
@@ -10,6 +10,21 @@
 
 user_token_address1 = make_ethereum_address()
 user_token_address2 = make_ethereum_address()
+
+MAKER_ASSET = EvmToken.initialize(
```

Comment:
no man wtf is this shit? I already explained it before and @nebolax had understood and already did it for all other tests!

We don't create constant assets in the tests like this. This is only used in `INITIAL_TOKENS` yes? Then turn `INITIAL_TOKENS` into a function `create_initial_globaldb_test_tokens` and in there initialize it. And call them when needed to be created (aka properly resolved).

This is what Alexey has already done in the rest of the tests.